config.py.sample

```diff
@@ -57,3 +57,6 @@ MEMCACHED_NAMESPACE = "AB"
 #MAIL_FROM_DOMAIN = "acousticbrainz.org"
 
 FILE_STORAGE_DIR = "./files"
+
+# Feature Flags
+FEATURE_EVAL_LOCATION = False
```

Comment:
ahh, good idea. My idea was to make sure we always got it from the config file with a default if it didn't exist, but this is good